tests/test-whitespace.sh

```diff
@@ -9,7 +9,7 @@ EOF
 
 export ADDITIONAL_INSTALL_STEPS='
 # add a space in paths and wrap in double-quotes
-RUN find /var/lib/tests -name "*.sh" -exec sed -i -E "s|/tmp/test([0-9]+)|\"/tmp/test \1\"|g" {} \;
+RUN find /var/lib/tests -name "*.sh" -exec sed -i -E "s|/tmp/test([0-9.]+)|\"/tmp/test \1\"|g" {} \;
```

Comment:
This shouldn't be necessary, `/tmp/test109 .1/something` should be a valid path.
On that note, why 109 and not 116 ? :)